includes/class-migration.php

```diff
@@ -136,6 +136,10 @@ public static function maybe_migrate() {
 			self::migrate_from_2_6_0();
 		}
 
+		if ( version_compare( $version_from_db, '4.0.0', '<' ) ) {
+			self::migrate_to_4_0_0();
```

Comment:
Will this logic run for a new install? Does it need to be added to `add_default_settings()`?

Reply:
Good point. I checked the code and in the case of a new install, it simply falls back to the default which is `0` and shows all IDs in the new format.